### ISSUE TITLE
Figure: MultiPLIER / DeepProfile - Rare Disease Putting It Together

### ISSUE THREAD

**Issue by cgreene** [jaybee84/ml-in-rd#108]
It'd be grand to have a figure that covers perhaps both of these topics showing how they incorporate regularization (PLIER constraints, VAE's loss function w/ KL-divergence), & prior data (MultiPLIER/DeepProfile)/knowledge(MultiPLIER).

**Comment by jaclyn-taroni** [comment]
I'm going to assign myself to this. I'll plan to put together at least an initial sketch in tandem with #111.

**Comment by jaclyn-taroni** [comment]
My plan for this is going to be to make a hand-drawn sketch to post here and also post the notes about the main takeaways we hope to show with the figure.

**Comment by jaclyn-taroni** [comment]
The text related to this figure is in the open pull request (#120). It can be viewed here: https://github.com/jaybee84/ml-in-rd/blob/jaclyn-taroni/111-all-together-now/content/06.multiple-approaches-required.md. I would recommend reading that as background.

Concepts covered in this section:

* **Regularization** (see also #106 and [this section of the manuscript](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/content/04.model-complexity.md), though we use it differently in the multiple approaches section hence the quote below!)

> In addition to reducing model complexity as discussed earlier, regularization can aid in useful representations by putting some constraints on what is learned by a model.

* **Transfer learning** - see also: https://github.com/jaybee84/ml-in-rd/issues/107#issuecomment-723699252
* **Representation learning** - see also [this section of the manuscript](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/content/03.heterogeneity.md#dimensionality-reduction-and-representation-learning), which I am quoting from below:

> Representation learning is the process of learning features from raw data, where a feature is an individual variable.

> Representation learning tends to be data-intensive (i.e. many samples are required) and thus may seem to aggravate the curse of dimensionality. But representation learning when applied to learn low dimensional patterns from large datasets and then applying those patterns to smaller but related datasets can be a powerful tool for dimensionality reduction. In the later sections of this perspective, we will discuss this method of leveraging large datasets to reduce dimensionality in smaller datasets, also known as _feature-representation-transfer_.

**Comment by jaclyn-taroni** [comment]
As with #107, I'm adding a sketch intended to communicate the main takeaways, but not necessarily guide the generation of the final figure.

I think the _big takeaway_ for this figure should be that the concepts/approaches discussed throughout don't get used in isolation; for things to work in this domain, you need multiple techniques!

![multiple-approaches](https://user-images.githubusercontent.com/19534205/98494353-caa8a780-220a-11eb-9a3b-5703c7c46971.jpg)

I'm again wary about getting into the details of either method in this figure, but I think showing a bunch of -> is also not entirely accurate. I think there's room for collapsing this to a single panel where we highlight differences between the approaches (e.g., what's the model, training data, what do we do with the output) perhaps through color scheme? We could even go more generic than that and not highlight the individual model differences in service of the main takeaway from above ☝️ 

> the concepts/approaches discussed throughout don't get used in isolation; for things to work in this domain, you need multiple techniques!

**Comment by allaway** [comment]
I like this sketch a lot. Maybe it is because I am more familiar with MultiPLIER and similar techniques than our target audience, but I think that this is an appropriate level of simplicity. I really like that this highlights how different inputs/methods (priors, regularization, transfer learning, etc) are used in practice in a couple of examples. I do not think that this needs to be made more generic, but that's just me.

**Comment by dvenprasad** [comment]
- I used the little bulb-gear icon to indicate that we were talking about a method. We can add that to the name of the method in the other figures too. ( also the bulb does not have to be smiling) 

**Deep Profile**
- For the last bit of the figure, is that improved VAE model or is it a different model we are generating?
- Also, what's the initial model health like? 

![deep-profile](https://user-images.githubusercontent.com/15315514/102669413-7de6b380-415c-11eb-8729-8be85699a7dc.png)

**MultiPLIER**
![multiplier](https://user-images.githubusercontent.com/15315514/102669414-7e7f4a00-415c-11eb-8372-e2522715b128.png)

**Comment by dvenprasad** [comment]
Updated MultiPLIER and DeepProfile based on comments from Monday's call. 

- Regularization and Representation learning happen at same step.
- Indicate that LVs and Encodings are representations
- Remove final model from DeepProfile

![multiplier](https://user-images.githubusercontent.com/15315514/103798789-33987a00-5018-11eb-94d3-729f41c67636.png)
![deep-profile](https://user-images.githubusercontent.com/15315514/103798790-34311080-5018-11eb-9856-cc98dcf48738.png)

**Comment by jaybee84** [comment]
@dvenprasad this looks good! Can we replace "Cell Line Data" with "In-vitro data"?
